lib/babel-wrap.js

```diff
@@ -0,0 +1,16 @@
+var path = require('path')
+
+// remove ourselves from process.argv
+process.argv = process.argv.filter(function (arg) {
```

Comment:
You're right; this was a leftover from trying to support it as a relative path, which was unfortunate to begin with.